### ISSUE TITLE
ESJsonLayoutTests fail on string comparison assertion

### ISSUE THREAD

**Issue by przemekwitek** [elastic/elasticsearch#44565]
FAILURE elastic/elasticsearch#7.x a5ad594 multijob-windows-compatibility/os=windows-2012-r2 - 20190718105648-DFE773DC

Recently introduced test (ESJsonLayoutTests) fails on Windows. Most likely this is due to new line characters being different:
```
14:40:38     java.lang.AssertionError: 
14:40:38     Expected: "{\"type\": \"server\", \"timestamp\": \"%d{yyyy-MM-dd'T'HH:mm:ss,SSSZZ}\", \"level\": \"%p\", \"component\": \"%c{1.}\", \"cluster.name\": \"${sys:es.logs.cluster_name}\", \"node.name\": \"%node_name\", \"message\": \"%notEmpty{%enc{%marker}{JSON} }%enc{%.-10000m}{JSON}\"%notEmpty{, %node_and_cluster_id }%exceptionAsJson }\n"
14:40:38          but: was "{\"type\": \"server\", \"timestamp\": \"%d{yyyy-MM-dd'T'HH:mm:ss,SSSZZ}\", \"level\": \"%p\", \"component\": \"%c{1.}\", \"cluster.name\": \"${sys:es.logs.cluster_name}\", \"node.name\": \"%node_name\", \"message\": \"%notEmpty{%enc{%marker}{JSON} }%enc{%.-10000m}{JSON}\"%notEmpty{, %node_and_cluster_id }%exceptionAsJson }\r\n"
```

Log: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+multijob-windows-compatibility/os=windows-2012-r2/15/console
Build Scans: https://gradle.com/s/rigslz4ooby6m, https://gradle.com/s/tuogchgpwby4a, https://gradle.com/s/au527lzzgydvq

**Comment by atorok** [comment]
This fails on master as well , I have an un-merged fix in b72be5cd004

**Comment by andrershov** [comment]
One more test failure, https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-windows-compatibility/os=windows-2012-r2/21/console.
To be fixed by @atorok commit.

**Comment by ebadyano** [comment]
A few more failures:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-windows-compatibility/os=windows-2016/27/consoleFull
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+multijob-windows-compatibility/os=windows-2019/23/consoleFull

**Comment by pgomulka** [comment]
fixed by #44675
